MdeModulePkg/Library/MemoryBucketLib/MemoryBucketLib.c

```diff
@@ -0,0 +1,530 @@
+/**@file
+Library for defining PEI memory buckets. This keeps track of the
+different buckets and the data for the associated HOB.
+For internal use only.

```

Comment:
Internal is a bit ambiguous here (i.e. "internal" to a company, package, driver, etc.).

Can you please add some more context?

Reply:
Updated to say the function shouldn't be called outside of the PEI CORE.